src/i18n/ru.json

```diff
@@ -75,7 +75,8 @@
       "CONFIRM_STOP": "Вы уверены, что хотите остановить виртуальную машину?",
       "CONFIRM_REBOOT": "Вы уверены, что хотите перезапустить виртуальную машину?",
       "CONFIRM_RESTORE": "Вы уверены, что хотите переустановить виртуальную машину?",
-      "REMOVE_SNAPSHOTS": "Пожалуйста, удалите снимки ВМ перед переустановкой",
+      "REMOVE_SNAPSHOTS_REINSTALL": "Пожалуйста, удалите снимки ВМ перед переустановкой",
```

Comment:
Do we need a period here after the sentence?

Reply:
@bwsw

Reply:
@andrewbents @wowshakhov that's fine as it is. I'll fix later if necessary.